AzureDb.Passwordless.MySqlConnectorTests/AzureDb.Passwordless.MySqlConnectorTests.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.8.2" />
```

Comment:
we should update to the latest 1.9.0

Reply:
latest stable release as of now seems 1.8.2

Reply:
oh, right - 1.9.0 is getting released this month.